pkg/sql/delegate/show_table.go

```diff
@@ -91,6 +91,11 @@ SELECT
 				ELSE
 					e'\n-- Warning: Partitioned table with no zone configurations.'
         END
+        ,
+        /* TODO(janexing): How to avoid printing the contraints that's already in the create stmt */
+        CASE WHEN (SELECT string_agg(details, E'\n') FROM [SHOW CONSTRAINTS FROM %[7]s] WHERE validated='f' AND constraint_type != 'PRIMARY KEY') IS NOT NULL THEN 
+            concat(E'\n', (SELECT string_agg(details, E'\n') FROM [SHOW CONSTRAINTS FROM %[7]s] WHERE validated='f' AND constraint_type != 'PRIMARY KEY'), ' NOT VALID') 
+        END
```

Comment:
I don't think this is the place to make the change. You should change the logic which generates the `CREATE TABLE` statement rather that doing anything up here in SQL. Also, you can (and probably should) have two separate PRs for these two changes.